genesis-volatility/src/config.ts

```diff
@@ -0,0 +1,15 @@
+import { logger } from '@chainlink/external-adapter'
+import { util } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/types'
+
+export const ENV_API_KEY = 'API_KEY'
+
+export const makeConfig = (prefix = ''): Config => {
+  return {
+    apiKey: util.getEnv(ENV_API_KEY, prefix),
+  }
```

Comment:
We now have `Requester.getDefaultConfig` that will do this for us:

```suggestion
  return Requester.getDefaultConfig(prefix)
```